doc/spec/design.md

```diff
@@ -695,6 +695,25 @@ Covers:
 
 Needs: impl, utest
 
+#### Full Coverage Tag Format With Needed Coverage
+`dsn~import.full-coverage-tag-with-needed-coverage~1`
+
+OFT imports coverage tags in the full tag format with a list of required/needed artifact types:
+
+    full-tag-with-needed-coverage = "[" *WSP reference "->" requirement-id ">>" artifact-type *("," *WSP artifact-type) "]"
+
+Rationale:
+
+The Tag importer is the catch all solution for all file formats that don't have a dedicated importer. Currently specification items imported via Tag importer can only be leaves in the specification tree. That means they can only cover but not require coverage.
```

Comment:
```suggestion
The Tag importer is the catch all solution for all file formats that don't have a dedicated importer. We want to allow specification items imported via Tag importer to be intermediate nodes in the specification tree, instead of limiting them to leaves in the specification tree. Because leaves can only cover other specification items, but not require coverage.
```